packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Added
+- SF-1185: Make a URL store
+  - `initHistory`, `pushState`, `updateHistory`, and `refreshState` methods on `flux`
+  - `updateHistory` action creator that triggers history `pushState` and `replaceState`, and updates the `history` section in the store
+  - `data.present.history` store section
+    - eg, ```
```

Comment:
The start of the code fence should be on a separate line for it to display properly. You can also add `js` to mark it as JavaScript.